aten/src/ATen/native/sparse/cuda/SparseCsrTensorMath.cu

```diff
@@ -347,8 +348,20 @@ Tensor _sparse_csr_..._cuda(const Tensor& input, IntArrayRef dims_to_sum, bool k
 
 namespace {
 
-template <typename scalar_t, typename index_t, typename ReductionOp>
-__global__ void reduce_sparse_csr_dim0_cuda_kernel(scalar_t* new_values,
+template <typename scalar_t, typename acc_t>
```

Comment:
are `new_values` and `new_values_acc` used at the same time ? If not, can we just have only ONE output for this `create_acc_buffer` ?

Reply:
`return std::move(input_);` will cause build error `moving a local object in a return statement prevents copy elision`.